release-notes/opensearch-release-notes-1.3.3.md

```diff
@@ -0,0 +1,41 @@
+# OpenSearch and OpenSearch Dashboards 1.3.3 Release Notes
+
+## Release Details
+
+OpenSearch Dashboards 1.3.3 includes the following features, enhancements, bug fixes, infrastructure, documentation, maintenance, and refactoring updates.
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-1.3.3.md).
+
+## Enhancements
+
+### Opensearch Anomaly Detection
+* Use current time as training data end time  ([#547](https://github.com/opensearch-project/anomaly-detection/pull/547))
+* Support writing features using filter aggregation ([#425](https://github.com/opensearch-project/anomaly-detection/pull/425))
+
+### Opensearch Ml Common
```

Comment:
The name should be `Ml Commons`

Reply:
Thanks @zelinh, pushed in my latest commit.